lib/containers/common.pm

```diff
@@ -242,6 +242,23 @@ sub test_search_registry {
     }
 }
 
+sub test_update_cmd {
+    my %args = @_;
+    my $runtime = $args{runtime};
+    my $container = $args{container};
+
+    # podman update was added to v4.3.0
+    if ($runtime eq 'podman') {
+        my $version = script_output "podman version | awk '/^Version:/ { print \$2 }";
+        return if (package_version_cmp($version, '4.3.0') <= 0);
+    }
+
+    my $cpu_shares = script_output "$runtime container inspect -f '{{.HostConfig.CpuShares}}' $container";
+    assert_script_run "$runtime update --cpu-shares 512 $container";
+    my $out = script_output "$runtime inspect -f '{{.HostConfig.CpuShares}}' $container";
+    die "$cpu_shares != $out" if ($cpu_shares != $out);
```

Comment:
It would be more straight forward to name the variables that somehow point the user to `default` and `newly` set values.
Syntax wise, it would be nice to also use the very same commands. At least we won't need to install additional tools as `jq`.

Reply:
> It would be more straight forward to name the variables that somehow point the user to `default` and `newly` set values.

The default for CpuShares is 0 but I was thinking of something like `$cpu_shares + 2` in case the default is changed to 512.

> Syntax wise, it would be nice to also use the very same commands. At least we won't need to install additional tools as `jq`.

Yeah.  I was using `jq` for the first command too until I remembered the `-f` option and only changed the first. I force-pushed a change.

Reply:
> > The default for CpuShares is 0 but I was thinking of something like `$cpu_shares + 2` in case the default is changed to 512. But by adding more settings we can be certain the feature works.
> 
> I guess, that would be better than relying on some constant assignment. However, for future reviews, it would be also nice to denote which variable holds the default value and which is the new one. Both are `cpu_shares` and `out`, that is not really self-explanatory

That will change when I add more parameters.  I plan to use a dictionary to track the command line option names and the JSON parameters.

My main issue is if you agree with me adding the test to `test_container_image()` to reuse the logic that downloads an image and start a container.

Reply:
Unless it is proven otherwise, I do not see objections with adding the test into `test_container_image()`